File: answers/Jatin_Bhargava/Conditionals/Question4.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
    int x,y;
    cin >> x >> y;
    if(x>0 && y>0) //+x,+y
    cout << "Quadrant 1";
    else if(x<0 && y<0) //-x,-y
    cout << "Quadrant 3";
    else if(x>0 && y<0) // +x,-y
    cout << "Quadrant 4";
    else if(x<0 && y>0) //-x,+y
    cout << "Quadrant 3";
    else if(x==0 && y==0)
    cout << "Origin";
    else if(x==0 && (y>0 || y<0))
    cout << "Y-axis";
    else if((x>0 || x<0) && y==0)
    cout << "X-asix";
} 

File: answers/AkhandAgrawal/Conditionals/question 4.py
x, y = input().split()
x = int(x)
y = int(y)
if x > 0 and y > 0:
    print("Quadrant 1")
elif x < 0 and y > 0:
    print("Quadrant 2")
elif x < 0 and y < 0:
    print("Quadrant 3")
elif x > 0 and y < 0:
    print("Quadrant 4")
elif x == 0:
    print("Y axis")
elif y == 0:
    print("X axis")
elif x == 0 and y == 0:
    print("Origin")
